File: rebus-rebuscrus/src/java/it/csi/rebus/rebuscrus/vo/UtilizzoVO.java
/*
* SPDX-FileCopyrightText: (C) Copyright 2022 Regione Piemonte
* SPDX-License-Identifier: EUPL-1.2
*/
package it.csi.rebus.rebuscrus.vo;

@SuppressWarnings("serial")
public class UtilizzoVO extends ParentVO {
	private	ContrattoProcVO contratto;
	private TipoContrattoVO tipoContratto;
	
	public ContrattoProcVO getContratto() {
		return contratto;
	}
	public void setContratto(ContrattoProcVO contratto) {
		this.contratto = contratto;
	}
	public TipoContrattoVO getTipoContratto() {
		return tipoContratto;
	}
	public void setTipoContratto(TipoContrattoVO tipoContratto) {
		this.tipoContratto = tipoContratto;
	}

	
}

File: rebus-rebuscruswcl/src/app/vo/utilizzo-vo.ts
/*************************************************
Copyright Regione Piemonte - 2022
SPDX-License-Identifier: EUPL-1.2-or-later
***************************************************/

import { ContrattoProcDatiVO, ContrattoProcVO } from "./contratto-proc-vo";
import { TipoContrattoVO } from "./extend-vo";

export class UtilizzoVO {
    contratto: ContrattoProcVO;
    tipoContratto: TipoContrattoVO;
    datiContratto: ContrattoProcDatiVO;
}